update_assets.py

```diff
@@ -10,78 +10,95 @@
 
 
 ASSETS = {
-    'bootstrap': {
-        'repo': 'twbs/bootstrap',
+#    'bootstrap': {
+#        'repo': 'twbs/bootstrap',
+#        'files': [
+#            {
+#                'src': 'scss',
+#                'dest': '_sass/bootstrap'
+#            },
+#            {
+#                'src': 'dist/js/bootstrap.min.js',
+#                'dest': '_includes/js/bootstrap.js'
+#            }
+#        ]
+#    },
+#    'font-awesome': {
+#        'repo': 'FortAwesome/Font-Awesome',
+#        'asset_id': 1,
+#        'files': [
+#            {
+#                'src': 'scss',
+#                'dest': '_sass/font-awesome'
+#            },
+#            {
+#                'src': 'webfonts',
+#                'dest': 'assets/webfonts'
+#            },
+#        ]
+#    },
+#    'leaflet': {
+#        'repo': 'Leaflet/Leaflet',
+#        'files': [
+#            {
+#                'src': 'dist/leaflet.css',
+#                'dest': '_sass/leaflet/leaflet.css'
+#            },
+#            {
+#                'src': 'dist/leaflet.js',
+#                'dest': '_includes/js/leaflet.js'
+#            }
+#        ]
+#    },
+#    'leaflet-easybutton': {
+#        'repo': 'CliffCloud/Leaflet.EasyButton',
+#        'files': [
+#            {
+#                'src': 'src/easy-button.css',
+#                'dest': '_sass/leaflet/leaflet-easybutton.css'
+#            },
+#            {
+#                'src': 'src/easy-button.js',
+#                'dest': '_includes/js/leaflet-easybutton.js'
+#            }
+#        ]
+#    },
+#    'leaflet-locatecontrol': {
+#        'repo': 'domoritz/leaflet-locatecontrol',
+#        'files': [
+#            {
+#                'src': 'dist/L.Control.Locate.min.css',
+#                'dest': '_sass/leaflet/leaflet-locatecontrol.css'
+#            },
+#            {
+#                'src': 'dist/L.Control.Locate.min.js',
+#                'dest': '_includes/js/leaflet-locatecontrol.js'
+#            }
+#        ]
+#    },
+#    'leaflet-providers': {
+#        'repo': 'leaflet-extras/leaflet-providers',
+#        'files': [
+#            {
+#                'src': 'leaflet-providers.js',
+#                'dest': '_includes/js/leaflet-providers.js'
+#            }
+#        ]
+#    },
```

Comment:
Could you please remove the comments to better see what changed in the file?

Reply:
This should not have been committed :D It was to prevent github from blacklisting me after executing the script a few times.